demos/list.scss

```diff
@@ -75,6 +76,22 @@ a.material-icons {
   margin-left: 72px;
 }
 
+.checkbox-list-ripple-surface {
+  @include mdc-ripple-surface;
+  @include mdc-ripple-radius;
+  @include mdc-states;
+  @include mdc-states-base-color(black);
+
+  position: relative;
+  outline: none;
```

Comment:
Is `outline: none` still necessary? The list item is no longer focusable.

Reply:
nope redundant.

Reply:
I realize I dont even need these mixins either...removing